integration/commands_administration_test.go

```diff
@@ -204,6 +204,71 @@ func TestCommandsAdministrationListCollections(t *testing.T) {
 	assert.Equal(t, compat, target)
 }
 
+func TestCommandsAdministrationCollectionUUID(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+	db := collection.Database()
+	collName := collection.Name()
+
+	err := db.CreateCollection(ctx, collName)
+	require.NoError(t, err)
+
+	var collUUID types.Binary
+
+	t.Run("ValidUUID", func(t *testing.T) {
+		cursor, err := db.ListCollections(ctx, bson.D{})
+		require.NoError(t, err)
+
+		var res []bson.D
+		err = cursor.All(ctx, &res)
+		require.NoError(t, err)
+		require.Len(t, res, 1)
+
+		doc := ConvertDocument(t, res[0])
+
+		path := types.NewStaticPath("info", "uuid")
+		uuid, err := doc.GetByPath(path)
+		require.NoError(t, err)
+		require.IsType(t, types.Binary{}, uuid)
+
+		collUUID = uuid.(types.Binary)
+		require.Len(t, collUUID.B, 16)
+		require.Equal(t, collUUID.Subtype, types.BinaryUUID)
+	})
+
+	// collection rename should not change the initial UUID
+	t.Run("CollectionRename", func(t *testing.T) {
```

Comment:
@AlekSi Since we have `-shuffle=on` for integration test, there is chance of this (`CollectionRename`) running before `ValidUUID` sub-test?

Reply:
Since these two sub-tests are dependent on each other, I'm thinking of making this as a single test case which  checks for both `ValidUUID` and `CollectionRename` cases. WDYT?

Reply:
Sure

Reply:
@AlekSi I have refactored the test case. PTAL.